quartz/components/ContentMeta.tsx

```diff
@@ -2,18 +2,37 @@ import { formatDate, getDate } from "./Date"
 import { QuartzComponentConstructor, QuartzComponentProps } from "./types"
 import readingTime from "reading-time"
 
-export default (() => {
+interface ContentMetaOptions {
+  /**
+   * Whether to display reading time
+   */
+  hideReadingTime: boolean
+}
+
+const defaultOptions: ContentMetaOptions = {
+  hideReadingTime: false,
+}
+
+export default ((opts?: Partial<ContentMetaOptions>) => {
+  // Merge options with defaults
+  const options: ContentMetaOptions = { ...defaultOptions, ...opts }
+
   function ContentMetadata({ cfg, fileData, displayClass }: QuartzComponentProps) {
     const text = fileData.text
+
     if (text) {
       const segments: string[] = []
-      const { text: timeTaken, words: _words } = readingTime(text)
 
       if (fileData.dates) {
         segments.push(formatDate(getDate(cfg, fileData)!))
       }
 
-      segments.push(timeTaken)
+      // Hide reading time if enabled
+      if (options.hideReadingTime == false) {
```

Comment:
prefer positives instead of negatives (let's rename it `options.showReadingTime` and default it to true